modules/test/playwright/helpers/ObjectAdminApiHelper.ts

```diff
@@ -16,6 +16,7 @@ import {ApiHelpers} from './ApiHelpers';
 
 export interface CreateObjectField {
 	attachmentSource?: string;
+	formulaFieldOutput?: 'Integer' | 'Decimal';
```

Comment:
can we change the order here to:
`formulaFieldOutput?:  'Decimal' | 'Integer';`